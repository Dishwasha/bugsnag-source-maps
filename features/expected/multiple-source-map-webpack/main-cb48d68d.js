(self.webpackChunkmultiple_source_map_webpack=self.webpackChunkmultiple_source_map_webpack||[]).push([[949],{945:()=>{console.log("hi from a")}}]);
//# sourceMappingURL=main-cb48d68d.js.map
