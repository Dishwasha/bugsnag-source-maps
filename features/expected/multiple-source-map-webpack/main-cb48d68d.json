{
    "version": 3,
    "sources": [
        "dist/webpack:/multiple-source-map-webpack/src/lib/a.js"
    ],
    "names": [
        "console",
        "log"
    ],
    "mappings": "sHAAAA,QAAQC,IAAI",
    "file": "main-cb48d68d.js",
    "sourcesContent": [
        "console.log('hi from a')\n"
    ],
    "sourceRoot": ""
}
